qc_schema/dev/properties/scf_properties.py

```diff
@@ -0,0 +1,71 @@
+"""
+The complete list of SCF level properties.
+"""
+
+scf_properties = {}
+
+scf_properties["scf_one_electron_energy"] = {
+    "description": "The one-electron energy contribution to the total SCF energy.",
+    "type": "number"
+}
+
+scf_properties["scf_two_electron_energy"] = {
+    "type": "number",
+    "description": "The two-electron energy contribution to the total SCF energy."
+}
+
+scf_properties["nuclear_repulsion_energy"] = {
+    "type":
+    "number",
+    "description":
+    r"""
+The nuclear repulsion energy contribution to the total SCF energy.
+
+.. math:: E_{NN} = \sum_{i, j<i}^{N_{atom}}\frac{Z_i Z_j}{|\mathbf{R}_i - \mathbf{R}_j|}
+"""
+}
+
+scf_properties["scf_vv10_energy"] = {
+    "type": "number",
+    "description": "The VV10 functional energy contribution to the total SCF energy."
+}
+
+scf_properties["scf_xc_energy"] = {
+    "type": "number",
+    "description": "The functional energy contribution to the total SCF energy."
+}
+
+scf_properties["scf_dispersion_correction_energy"] = {
+    "type":
+    "number",
+    "description":
+    """
+The dispersion correction appended to an underlying functional
+when a DFT-D method is requested.`
+"""
+}
+
+scf_properties["scf_dipole_moment"] = {
```

Comment:
Does this have to be 'scf' dipole moment? Maybe I'm pedantic, but what if I run CCSD(T) - most programs don't print out two different dipole moments IIRC.

Reply:
I think its good to clarify if someone does say CCSD bond orders. It can become unclear if you stored the SCF or CCSD dipoles.

Reply:
Won't that lead to a proliferation of new objects? Maybe there should be a `properties` object:
```
property: {
  type: "dipole moment"
  method: "SCF"
  value: ...
}
```

Reply:
It doesn't seem like too many more fields as we will have energy definitions for these slots anyways. I might be missing your point on the object side of things however.